### PR TITLE
Document JSON API

### PR DESCRIPTION
**Pull request by phosei**
Once this is merged, I’d like to make a new release. 🎉

### CODE REVIEW

**Review by j-frost — APPROVED** [review]
Does the supplied data have to be an object with a member called data that is of type string? If so we could clarify that.

File: README.md [hunk 1]
```diff
@@ -37,3 +37,15 @@ Use the following environment variables:
...
+API
+---
+
+Post JSON to `/new` to create a new secret programmatically:
+
+    $ curl -X POST https://my.ff/new -H 'Content-Type: application/json' \
+        --data '{ "data": "hello world" }'
+    {"secret_link":"https://my.ff/get/foo/bar","status":"ok"}
+
+As you can see, you’ll get a JSON response containing the secret link.
```
Comment (fkusei) [inline README.md:53]: `status` may also be `"error"`, in which case there's a `msg` containing more info.

**Comment by phosei** [comment]
Hey, I tried to address your comments. 🙇

**Review by fkusei — APPROVED** [review]

**Comment by j-frost** [comment]
I like the changes.

**Pull request merged**